build/.vsts-ci.yml

```diff
@@ -966,3 +966,23 @@ jobs:
       cd samples/dockerbuilds/MIPS32
       docker build -t mipsiotbuild:latest . --network=host
     displayName: 'Build MIPS32'
+- job: crosscompile_arm
+  pool:
+    vmImage: 'ubuntu-20.04'
+  displayName: "Cross Compile (ARM)"
+  steps:
+  - script: |
+      sudo apt-get update && sudo apt-get install -y \
+      curl \
+      git \
+      python-software-properties \
+      build-essential \
+      pkg-config \
+      docker.io
+      sudo curl -sL https://deb.nodesource.com/setup_16.x | bash -
+      sudo apt-get install -y nodejs
+    displayName: 'Setup'
+  - script: |
+      cd samples/dockerbuilds/ARM
+      docker build -t armiotbuild:latest . --network=host
```

Comment:
I think you then need to build with the adjunct file after this?

Reply:
No, the adjunct file uses "myapp" for building. The documentation shows how to use adjunct file to build one of our samples.

Reply:
Does the Docker daemon take in both files and execute them both? I thought it would only take `Dockerfile` and otherwise, you have to specify the file with `-f`. Such as

`docker build -f Dockerfile_adjunct -t myapp:latest`

Ultimately, I'm curious how the adjunct file gets used here since that's the one that seems to be doing the cross compiling.

Reply:
here is the docs https://github.com/Azure/azure-iot-sdk-c/blob/main/doc/Docker_SDK_Cross_Compile.md

Reply:
right now we only do the first part

# Cross compile the SDK
docker build -t mipsiotbuild:latest ./MIPS32 --network=host

Reply:
do you think there is any value in build the Dockerfile_adjunct? We could add 

`cd samples/dockerbuilds`
`docker build -t armiotapp:latest . --network=host --file ./ARM/Dockerfile_adjunct`

Reply:
Yes because it looks like that's where the actual compilation with the new toolchaine is done:

https://github.com/Azure/azure-iot-sdk-c/blob/cc2f4ef1347f76e12197d8c2b921a695bd2cb46e/samples/dockerbuilds/ARM/Dockerfile_adjunct#L20-L21